@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sfmplus/fn_sfmplusStabilator.sqf

```diff
@@ -0,0 +1,152 @@
+//    A-------------H-------------B
+//    |             |             |
+//    E-------------G-------------F
+//    |             |             |
+//    D-------------I-------------C
+private _width  = _heli getVariable "fza_ah64d_stabWidth";
+private _length = _heli getVariable "fza_ah64d_stabLength";
+
+private _halfWidth = _width / 2;
+
+private _A = _stabPvt vectorAdd [-_halfWidth, 0, 0];
+private _B = _stabPvt vectorAdd [_halfWidth, 0, 0];
+private _C = _stabPvt vectorAdd [_halfWidth, cos _theta * (-_length), sin _theta * _length];
+private _D = _stabPvt vectorAdd [-_halfWidth, cos _theta * (-_length), sin _theta * _length];
+
+private _E = (_A vectorAdd _D) vectorMultiply 0.5;
+private _F = (_B vectorAdd _C) vectorMultiply 0.5;
+private _G = (_E vectorAdd _F) vectorMultiply 0.5;
+
+private _H = (_A vectorAdd _B) vectorMultiply 0.5;
+private _I = (_D vectorAdd _C) vectorMultiply 0.5;
+
+private _liftLine  = _E vectorDiff _F;
+private _chordLine = _H vectorDiff _I;
+
+private _liftVec = vectorNormalized (_chordLine vectorCrossProduct _liftLine);
+_liftVec = _liftVec;
+
+private _stabLine = vectorNormalized _chordLine;
+_stabLine = _stabLine;
+
+private _relWind = vectornormalized(_heli vectorWorldToModel (velocity _heli));
+_relWind = _relWind;
+
+private _AoA = (_relWind # 2 atan2 _relWind # 1) + _theta;
+_AoA = [_AoA] call CBA_fnc_simplifyAngle180;
+
+private _AIRFOILTABLE =
+[
+//------AoA-[0]----------CL-[1]--------CD-[2]-------------
+    [   -180.0,        0.0,           0.0        ],  //0  - DO NOT CHANGE!!
+    [   -135.0,        0.5,           0.5        ],  //1  - DO NOT CHANGE!!
+    [   -90.0,         0.0,           0.0        ],  //2  - DO NOT CHANGE!!
+    [   -18.5,         -1.22580,      0.10236    ],  //3  -
+    [   -17.5,         -1.30310,      0.07429    ],  //4  -
+    [   -15.75,        -1.38680,      0.03865    ],  //5  -
+    [   -10.0,         -1.08070,      0.01499    ],  //6  -
+    [   -5.0,          -0.55710,      0.00847    ],  //7  -
+    [   0.0,           0.00000,       0.00540    ],  //8  -
+    [   5.0,           0.55720,       0.00847    ],  //9  -
+    [   10.0,          1.08080,       0.01499    ],  //10 -
+    [   15.75,         1.38810,       0.03863    ],  //11 -
+    [   17.5,          1.30590,       0.07416    ],  //12 -
+    [   18.5,          1.22840,       0.10229    ],  //13 -
+    [   90.0,          0.0,           0.0        ],  //14 - DO NOT CHANGE!!
+    [   135.0,         -0.5,          -0.5       ],  //15 - DO NOT CHANGE!!
+    [   180.0,         0.0,           0.0        ]   //16 - DO NOT CHANGE!!
+];
+
+private _intAIRFOILTABLE = [_AIRFOILTABLE, _AoA] call fza_fnc_linearInterp;
+private _CL = _intAIRFOILTABLE select 1;
+
+private _area = [_A, _B, _C, _D] call fza_fnc_sfmplusGetArea;
+
+private _liftForce = -_CL * 0.5 * 1.225 * _area * (_V_mps * _V_mps);
+
+private _lift = _liftVec vectorMultiply (_liftForce * _deltaTime);
+_heli addForce[_heli vectorModelToWorld _lift, _G];
+
+/*
```

Comment:
We could throw the debug stuff in an ifdef `__A3_DEBUG__` so that it only shows in debug mode.

```sqf
#ifdef __A3_DEBUG__
//Code goes here
#endif
```